docs-v2/source/includes/_configuration.md

```diff
@@ -136,6 +140,21 @@ When the`directToCheckoutMode` option is set to `true`, after clicking the `buy
 
 ***
 
+### Add To Cart Popup
+
+```javascript
+const tipserOptions={
+ addToCartPopup: {
+    className: "custom-className",
+    duration: 5000;
+  }
+};
+```
+- `className` parameter adds custom className to popup component.
```

Comment:
"parameter adds custom className to popup component." -> adds a custom class name to the "added to cart" popup

"parameter sets the time the popup appears (in ms)." -> the amount of time (in ms) the "added to cart" popup remains visible